msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/AcquireTokenB2CTest.java

```diff
@@ -40,17 +41,34 @@ public String getAuthority() {
         return mApplication.getConfiguration().getDefaultAuthority().getAuthorityURL().toString();
     }
 
-    @Override
-    public String getConfigFilePath() {
-        return B2C_CONFIG_FILE_PATH;
-    }
-
     @Override
     public String[] getScopes() {
         return B2C_SCOPE;
     }
 
-    public static class B2CLocalUser extends AcquireTokenB2CTest {
+    public static class B2CLocalUserGlobalMsftDomain extends AcquireTokenB2CTest {
+
+        @Override
+        public String getConfigFilePath() {
+            return B2C_CONFIG_FILE_PATH;
```

Comment:
nit: rename this to reflect the class name (i.e. same as what you did with B2C_CUSTOM_DOMAIN_CONFIG_FILE_PATH).

Reply:
Addressed